pkg/minikube/config/types.go

```diff
@@ -54,18 +54,19 @@ type MachineConfig struct {
 
 // KubernetesConfig contains the parameters used to configure the VM Kubernetes.
 type KubernetesConfig struct {
-	KubernetesVersion string
-	NodeIP            string
-	NodeName          string
-	APIServerName     string
-	APIServerNames    []string
-	APIServerIPs      []net.IP
-	DNSDomain         string
-	ContainerRuntime  string
-	NetworkPlugin     string
-	FeatureGates      string
-	ServiceCIDR       string
-	ExtraOptions      util.ExtraOptionSlice
+	KubernetesVersion   string
+	NodeIP              string
+	NodeName            string
+	APIServerName       string
+	APIServerNames      []string
+	APIServerIPs        []net.IP
+	DNSDomain           string
+	ContainerRuntime    string
+	NetworkPlugin       string
+	FeatureGates        string
+	KubeadmFeatureGates string
```

Comment:
Would it be possible to reuse the existing FeatureGates field here instead of adding a new one?

Reply:
Yes, I will remove it and reuse FeatureGates field.

Reply:
@dlorenc  I have been updated, could you review this changes? thanks!

Reply:
ping.